linalg/blockoperator.cpp

```diff
@@ -246,7 +246,7 @@ BlockLowerTriangularPreconditioner::BlockLowerTriangularPreconditioner(
      offsets(0),
      ops(nBlocks, nBlocks)
 {
-   ops = static_cast<Operator *>(NULL);
+   ops = static_cast<Operator *>(nullptr);
```

Comment:
```suggestion
   ops = nullptr;
```

Reply:
removed in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7